docs/Java-SpringBoot-Developer.md

```diff
@@ -11,6 +11,9 @@
     - Work in progress
```

Comment:
Based on the discussion in the standup chat earlier, will need to update this section once the necessary fixes are made to java.